<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageTransform</string>
  </property>
  <property name="windowIcon">
   <iconset resource="default.qrc">
    <normaloff>:/img/src/logo_2.png</normaloff>:/img/src/logo_2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="GraphicsView" name="leftGraphicsView" native="true"/>
        </item>
        <item>
         <widget class="GraphicsView" name="rightGraphicsView" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1198</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuColor_Space">
    <property name="title">
     <string>control</string>
    </property>
    <addaction name="actionRGB2HSV"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuColor_Space"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/Open_folder_full_48px_1186194_easyicon.net.png</normaloff>:/img/src/Open_folder_full_48px_1186194_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/clear_gtk_72px_43338_easyicon.net.png</normaloff>:/img/src/clear_gtk_72px_43338_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/save.svg</normaloff>:/img/src/save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/save.svg</normaloff>:/img/src/save.svg</iconset>
   </property>
   <property name="text">
    <string>Sa&amp;ve As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRGB2HSV">
   <property name="icon">
    <iconset resource="default.qrc">
     <normaloff>:/img/src/Final_Cut_Pro_X_96px_1134672_easyicon.net.png</normaloff>:/img/src/Final_Cut_Pro_X_96px_1134672_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>RGB2HSV</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QWidget</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="default.qrc"/>
 </resources>
 <connections/>
</ui>
